package.json

```diff
@@ -26,7 +27,8 @@
     "hubot-slack": "~3.3.0",
     "hubot-spotify-me": "~1.0.1",
     "hubot-youtube": "~1.0.0",
-    "request": "^2.65.0"
+    "request": "^2.65.0",
+    "time": "0.8.2"
```

Comment:
Hier tript ie (bij mij) op; als je er `"time": "0.11.4"` van maakt, start florriebot in ieder geval weer op :smiley: